File: CS/Q559000/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.Web.ASPxEditors;

namespace Q559000.Controllers
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/

        public ActionResult Index()
        {
            return View();
        }
        Q559000.Models.NorthwindDataContext db = new Q559000.Models.NorthwindDataContext();

        [ValidateInput(false)]
        public ActionResult GridViewPartial()
        {
            var model = db.Orders;
            return PartialView("_GridViewPartial", model);
        }

    }
}

File: VB/Q559000/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.Web.ASPxEditors

Namespace Q559000.Controllers
	Public Class HomeController
		Inherits Controller
		'
		' GET: /Home/

		Public Function Index() As ActionResult
			Return View()
		End Function
		Private db As New Q559000.Models.NorthwindDataContext()

		<ValidateInput(False)> _
		Public Function GridViewPartial() As ActionResult
			Dim model = db.Orders
			Return PartialView("_GridViewPartial", model)
		End Function

	End Class
End Namespace
